### ISSUE TITLE
How to create train, test and validation set?

### ISSUE THREAD

**Issue by AmoghM** [kaize0409/GCN_AnomalyDetection#3]
The model reports only the train loss and no validation loss is provided. Although the anomalies were added synthetically, therefore we know the ground truth and can use metrics of precision, recall to check how the model performs on validation data.

**Comment by yzfxmu** [comment]
> The model reports only the train loss and no validation loss is provided. Although the anomalies were added synthetically, therefore we know the ground truth and can use metrics of precision, recall to check how the model performs on validation data.

Have you solved this problem?
